libs/documentation/src/lib/storybook/formly/formly-stepper/formly-stepper.stories.ts

```diff
@@ -0,0 +1,162 @@
+import { ReactiveFormsModule, FormsModule } from '@angular/forms';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { SdsFormlyModule } from '@gsa-sam/sam-formly';
+import { FormlyModule } from '@ngx-formly/core';
+import { moduleMetadata, Meta, Story } from '@storybook/angular';
+import {
+  createCodePreviewTabData,
+  generateConfig,
+  generateStackblitzLink,
+} from 'libs/documentation/src/sandbox/sandbox-utils';
+import { FormlyStepperIntroductionModule } from './formly-stepper-introduction/formly-stepper-introduction.module';
+import { FormlyStepperBasicModule } from './formly-stepper-basic/formly-stepper-basic.module';
+import { FormlyStepperAdvancedModule } from './formly-stepper-advanced/formly-stepper-advanced.module';
+import { UswdsStepperModule } from './formly-stepper-uswds/formly-stepper-uswds.module';
+import { SdsStepper } from '@gsa-sam/sam-formly';
+import { RouterTestingModule } from '@angular/router/testing';
+
+const templateOptions = {
+  table: {
+    category: 'template-options',
+  },
+};
+
+export default {
+  title: 'Formly/Stepper',
+  decorators: [
+    moduleMetadata({
+      imports: [
+        CommonModule,
+        ReactiveFormsModule,
+        SdsFormlyModule,
+        FormsModule,
+        FormlyModule.forRoot(),
+        FormlyStepperIntroductionModule,
+        NoopAnimationsModule,
+        FormlyStepperBasicModule,
+        FormlyStepperIntroductionModule,
+        RouterTestingModule,
+        FormlyStepperAdvancedModule,
+        UswdsStepperModule,
+      ],
+      providers: [SdsStepper],
+    }),
+  ],
+  argTypes: {
+    label: templateOptions,
+    placeholder: templateOptions,
+    description: templateOptions,
+    required: templateOptions,
+    disabled: templateOptions,
+    tooltipText: templateOptions,
+  },
+} as Meta;
+
+export const Introduction: Story = (args) => ({
+  template: '<sds-formly-stepper-introduction></sds-formly-stepper-introduction>',
+  props: args,
+});
+Introduction.parameters = { options: { showPanel: false } };
+
+export const Uneven: Story = (args) => ({
+  template: '<sds-formly-stepper-basic></sds-formly-stepper-basic>',
+  props: args,
+});
+Uneven.parameters = {
+  controls: {
+    disabled: true,
+    hideNoControlsWarning: true,
+  },
+  actions: { disabled: true },
+  preview: generateConfig(
+    'storybook/formly/formly-stepper/formly-stepper-basic',
+    'FormlyStepperBasicModule',
+    'sds-formly-stepper-basic',
+    [
+      createCodePreviewTabData(
+        'storybook/formly/formly-stepper/formly-stepper-basic/formly-stepper-basic-uneven.component.ts',
+        'ts',
+        false
+      ),
+      createCodePreviewTabData(
+        'storybook/formly/formly-stepper/formly-stepper-basic/formly-stepper-basic-uneven.component.html',
+        'html',
+        false
+      ),
+    ]
+  ),
+  stackblitzLink: generateStackblitzLink('formly-stepper', 'basic'),
+};
+
+export const USWDS: Story = (args) => ({
```

Comment:
Stackblitz fails:
ERROR
Error: R3InjectorError(AppModule)[Router -> Router -> Router]:
NullInjectorError: No provider for Router!